### ISSUE TITLE
The model does not converge on my own dataset

### ISSUE THREAD

**Issue by weiyithu** [cv-rits/MonoScene#34]
Dear author,
   I am trying to train the model on another dataset. However, the loss is around 22 and degrades very slowly. Does the model converge fast on KITTI dataset? If possible, could you show a loss curve?

**Comment by anhquancao** [comment]
How is the metric? in my case, the validation loss increases but the metrics get better

**Comment by weiyithu** [comment]
Thanks for your responce. Maybe there is a bug in my code. As mentioned in #30, if I set T_velo_2_cam as np.eye(4), should I change the scene size (256, 256, 32) to (256, 32, 256)? Since the z axis in camera coordinate means depth.

**Comment by anhquancao** [comment]
Yes, you need to change the scene_size and scene_origin. Becareful also with the direction of the axis as sometimes the y axis is pointing down.

**Comment by weiyithu** [comment]
Thanks! I will check my code.

**Comment by hongxiaoy** [comment]
> Yes, you need to change the scene_size and scene_origin. Becareful also with the direction of the axis as sometimes the y axis is pointing down. 1 useful check is that you can draw your 3D scene and only show the voxels inside the FOV.

It seems that I am also faced with the coordinate problem, so is the target label shape (60, 36, 60) in NYU coresponds to (x, y, z) in camera frame, am I right? Or it coresponds to (x, y, z) in world frame, but NYU has a different world frame direction? Thanks.

**Comment by anhquancao** [comment]
Hi @hongxiaoy , for NYUv2, it is in the world frame.
